### ISSUE TITLE
Open also SVG in Browser

### ISSUE THREAD

**Issue by janstieler** [igorskyflyer/vscode-open-in-browser#18]
Hi,
it would be great if the plugin would also open SVGs in my browser.
Actuall I get the message `no active html file`.

cheers

**Comment by igorskyflyer** [comment]
Yes, the extension only recognises and works with HTML files (.xhtml, .htm and .html), I will expand the detection to SVG files as well, thanks. 😁
